frameworks/helloworld/src/main/dist/pod-profile-mount-volume.yml

```diff
@@ -28,7 +35,14 @@ pods:
       noprofile:
         goal: RUNNING
         essential: false
-        cmd: df -t $VOLUME_PROFILE pod-container-path >> pod-container-path/output && df -t ext4 noprofile-container-path >> noprofile-container-path/output && sleep $SLEEP_DURATION
+        cmd: >
+               set -e -x
+               [ -n "{{{TEST_PROFILE_VOLUME_COMMAND}}}" ]
+               {{{TEST_PROFILE_VOLUME_COMMAND}}} pod-container-path
+               ! {{{TEST_PROFILE_VOLUME_COMMAND}}} noprofile-container-path
```

Comment:
What is the purpose of the `!` here?

Reply:
Just saw your comment above.

Reply:
The `TEST_PROFILE_VOLUME_COMMAND` is used to test that a path is using a certain filesystem. For the existing test that is used to test that filesystem is using `xfs`. In the old code we were testing that `noprofile-container-path` is of type `ext4`. This doesn't work with the new approach. Hence, we're testing that we are NOT using `xfs` here.

Reply:
OK, I though that `!` meant something different in bash. Thanks for the clarification.